src/sentry/south_migrations/0368_backfill_grouphash_last_processed_event_data.py

```diff
@@ -0,0 +1,1034 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import itertools
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from sentry.utils.iterators import chunked
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
I'd imagine we will want to set this on one or both of these migrations?

Reply:
Yes please.

Reply:
Done in 313167d6d0ae7a174ac23ef7df6f934c70472ace. Let me know what you want to do about limiting the event queryset for the backfill.